### DISCUSSION TITLE
Is there any way to create a circle diagram instead of bar graph?

### DISCUSSION THREAD

**Discussion by shyneewalzer** [mathoudebine/turing-smart-screen-python#202] [Q&A] [answered]
I have read the documentation and says there are two types of metrics(GRAPH and TEXT), does the graph metrics really limited only to bar graphs? Are there any other tricks like to make some circular graph metric?

I am trying to make something like this
![image](https://user-images.githubusercontent.com/76527288/232181608-002d9cea-5fb6-4629-8c3c-d47aec831f5a.png)

I would appreciate it more if you know any work around, advanced thank you.

**Comment by mathoudebine** [comment, marked as answer]
Hello @shyneewalzer 
A bit late maybe but this is now implemented.
If you use the last release you can add "radial" bar into themes to do what you want.
Check the `LandscapeMagicBlue` theme as example!
